### DISCUSSION TITLE
@sugarat/theme 主题实现相关问题讨论

### DISCUSSION THREAD

**Discussion by ATQQ** [ATQQ/sugar-blog#78] [Ideas]
😄   主题相关的问题都可在这讨论（包含但不限于源码，功能，bug等等）

**Comment by JiansongLee** [comment]
您好，这个主题清爽简洁，很漂亮！是否有考虑评论插件增加可选项，比如waline?

**Reply by ATQQ** [reply]
这个我下来看一下嘞 理论上可以实现😁

**Reply by JiansongLee** [reply]
这个有一点好处，就是可以自建服务器。比较自由。

**Reply by JiansongLee** [reply]
前几天安装完这个主题后，有发现首页的友情链接、标签两个地方的emoji图标显示乱码。

**Reply by ATQQ** [reply]
@JiansongLee 稳定复现吗？能截个图吗

**Reply by JiansongLee** [reply]
可以稳定复现。比如下面这个图，是直接从你的博客主页看到的，和我安装后看到的一样。我用的是CHROME浏览器。
![无标题](https://github.com/ATQQ/sugar-blog/assets/12186504/6509b40b-2ee8-4bc7-8916-eea5b6c4d6a2)

**Reply by ATQQ** [reply]
<img width="116" alt="image" src="https://github.com/ATQQ/sugar-blog/assets/42485491/ec07b8a7-7200-4ce6-89e7-73697acb3515">
我这边ok的 应该是设备不支持这个 emoji 🤝

你那啥系统

**Reply by ATQQ** [reply]
这个 我默认换成 SVG了 应该没问题嘞，你可以再更新试一下 @JiansongLee

**Reply by JiansongLee** [reply]
赞！辛苦了。

**Comment by i-mufeng** [comment]
很喜欢你的主题，非常感谢大佬。
最近在使用过程中有一些小的建议：

### 首页部分

1. 希望可以在 `BlogHomeInfo` 中最上方加入一个头像，可选自定义链接或者 Gravatar 镜像。
2. 希望可以将页脚 `footer` 对象的 `message` 和 `copyright` 重写为 list，更适合中国宝宝体质，毕竟备案号啥的要求比较多。

我也在自己研究解决方法，但是能力有些不够。。。

**Reply by ATQQ** [reply]
没问题的，本周优先安排一下 哈哈

**Reply by i-mufeng** [reply]
Thank you!

**Reply by ATQQ** [reply]
@i-mufeng 已经可以啦，你可以更新主题，看一下最新的主题使用文档即可
 https://theme.sugarat.top/config/global.html#footer

**Reply by i-mufeng** [reply]
好嘞好嘞~

**Reply by i-mufeng** [reply]
@ATQQ 体验了一下，很喜欢，真的十分感谢。

有点小建议：这块头像和博文信息是有关联性的，头像这块没有 `card` 的样式感觉风格不统一，放到下方的 `card overview-data` 里会不会好一些。
![image](https://github.com/ATQQ/sugar-blog/assets/68867110/bdc9f766-884a-412e-b4b9-9b550018989f)

**Reply by ATQQ** [reply]
可以的 下周搞搞😉  

我再学习一下其它站点

**Reply by i-mufeng** [reply]
大佬，还有一点，就是我观察到诸如腾讯阿里，亦或者其他同款博客网站。他们的 `footer` 部分会将备案号等链接跟 `copyright` 分为两块来放，且样式不一定相同（备案号、友链等都是链接，`copyright` 不会是链接），但是博客目前是在一行，麻烦你评估一下是不是可以调整为两行放置。

![image](https://github.com/ATQQ/sugar-blog/assets/68867110/24e33e62-045c-4764-9248-10af2d463573)

![image](https://github.com/ATQQ/sugar-blog/assets/68867110/376e7dec-5f0d-49af-8f87-09cab043e64a)

![image](https://github.com/ATQQ/sugar-blog/assets/68867110/88947ebc-84c2-4a00-b72d-ac406cfb0afc)

**Reply by ATQQ** [reply]
目前提供了 message 属性的哦 可以自定义上面的内容

![image](https://github.com/ATQQ/sugar-blog/assets/42485491/85e81ade-bd55-4aa4-83d3-0a912bd75fa3)

应该是可以满足你的需要的

**Comment by i-mufeng** [comment]
大佬大佬，我有一个建议。

我觉得在竖屏状态下，应该是博主信息在上，文章列表在下。但是现在刚好相反，不知道你怎么看呢？

![image](https://github.com/ATQQ/sugar-blog/assets/68867110/fab8ba7f-642f-410a-8b42-2c14da6a0c9f)

**Reply by ATQQ** [reply]
这个顺序的话 emm 你可以截图看一下 我感觉直接放上去也不太好看，我下来找几个博客学习一下

![image](https://github.com/ATQQ/sugar-blog/assets/42485491/f79d46b4-0c23-4158-ba70-db9b5a4bb5cd)

**Comment by ATQQ** [comment]
@i-mufeng 上面提到的几个点都搞了一下，CHANGLOG: https://theme.sugarat.top/changelog.html#_0-2-4-2023-09-30

* 卡片样式（`avatarMode `）：https://theme.sugarat.top/config/frontmatter.html#blog
* 页脚更丰富的自定义：https://theme.sugarat.top/config/global.html#footer
* 小屏加了个头像目前（`minScreenAvatar `）：https://theme.sugarat.top/config/frontmatter.html#blog

**Reply by i-mufeng** [reply]
十分感谢作者大大，假期太忙没顾得上看邮件。
